addon/templates/components/polaris-empty-search-result.hbs

```diff
@@ -0,0 +1,25 @@
+{{#polaris-stack
+  alignment="center"
+  vertical=true
+}}
+  {{#if withIllustration}}
+    {{!-- TODO: replace this with polaris-image when implemented. --}}
+    <img
```

Comment:
This should be wrapped in an explicit `stack.item` to avoid a potential [glimmer error](https://github.com/smile-io/ember-polaris/pull/159) if `withIllustration` changes from truthy to falsy